cpp/src/jit/parser.cpp

```diff
@@ -387,52 +326,34 @@ std::string ptx_parser::parse()
 
   input_arg_list.clear();
   // Go directly to the .func mark
-  const size_t length = no_comments.size();
-  size_t start        = no_comments.find(".func") + 5;
-  if (start == length + 5) {
+  size_t offset = no_comments.find(".func");
+  if (offset == no_comments.size()) {
     printf("No function (.func) found in the input ptx code.\n");
     exit(1);
```

Comment:
Not a comment on this PR specifically, but isn't `exit(1)` kind of extreme here?

Reply:
I agree, let's see if people respond to the slack thread. I would think we could do in a follow up job if we want to change though.